maintenance/ebpf/ebpf-and-eks.md

```diff
@@ -233,7 +233,7 @@ kind: ConfigMap
 apiVersion: v1
```

Comment:
Looks right to me, there's also a reference on line 228 in the main text.  Please can you fix that up too?